pkg/controller/status/status.go

```diff
@@ -131,6 +135,10 @@ func (m *statusManager) Run() {
 				m.clearDegraded()
 			}
 
+			if !m.enabled {
```

Comment:
Looks like we have this in two places within this loop - why is that? Can we resolve these to a single check? 

I think you can just put this at the top of the loop?

```go
if !m.enabled {
   cleanup()
   time.Sleep( 5 * time.Second)
   continue
} 

if !m.syncState() {
 . . .
}
...
```